src/fsharp/LanguageFeatures.fs

```diff
@@ -62,12 +62,14 @@ type LanguageVersion (specifiedVersionAsString) =
             LanguageFeature.RelaxWhitespace, languageVersion47
             LanguageFeature.ImplicitYield, languageVersion47
 
+            // F# 5.0
+            LanguageFeature.FixedIndexSlice3d4d, languageVersion50
+            LanguageFeature.FromEndSlicing, languageVersion50
```

Comment:
Would this mean that specifying `preview` will not light up these features?

Reply:
That is what it looks like too. These features need to be in preview.

Reply:
@cartermp:    Preview lights up every feature that is assigned to a version and added to the table.

https://github.com/dotnet/fsharp/blob/0600685f4cf1711a3ce9b3aecd8ee4d4c6c86792/src/fsharp/LanguageFeatures.fs#L67

No they don't need to be in preview, since they are already approved for 5.0.  There is of course no way to specify langversion:5.0, however, since 5.0 is less than 9999 then all is good.

--langversion verification:
https://github.com/dotnet/fsharp/blob/0600685f4cf1711a3ce9b3aecd8ee4d4c6c86792/src/fsharp/LanguageFeatures.fs#L56

Reply:
> No they don't need to be in preview, since they are already approved for 5.0. 

I disagree, they should be in preview. We haven't had anyone use them and give feedback about their usage, which is the primary purpose of having the preview flag. Rather than establishing a cut line now, we should do that when we've had ample time for features like these, `nameof`, and others.